example/app/data/DemoDataSource/DemoPackage/entities/dummy_entity.json

```diff
@@ -1,6 +1,6 @@
 {
   "_id": "f5282220-4a90-4d02-8f34-b82255fc91d5",
   "type": "blueprints/DummyBlueprint",
-  "name": "dummyEntity",
+  "name": "recipesAsSidebar",
```

Comment:
maybe rename file to recipesAsSidebar.json?

Reply:
fixd